content/en/development/design-docs/blb-migration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Basic to Standard Load Balancer Migration"
+linkTitle: "Basic to Standard Load Balancer Migration"
+type: docs
+description: >
+    Support migration from Basic Load Balancer to Standard Load Balancer.
+---
+
+## Background
+
+On September 30, 2025, Basic Load Balancer will be [retired](https://learn.microsoft.com/en-us/azure/load-balancer/skus). This design document describes the planned migration path from Basic Load Balancer to Standard Load Balancer.
+
+## Design
+
+> The following `User` stands for human users, or any cluster provisioning tools (e.g., AKS).
+
+1. User must manually decouples all virtual machines from the Basic Load Balancer backend pool before migration.
+
+2. User must change the Load Balancer sku in the configuration file from `Basic` to `Standard`.
+
+3. If the basic Load Balancer is not removed, it will be deleted during the migration.
```

Comment:
if SLB outbound is required, then basic LB must be deleted and recreate with same name